### DISCUSSION TITLE
Where is the data?

### DISCUSSION THREAD

**Discussion by Blueforcer** [thijsvanloef/palworld-server-docker#25] [Q&A]
Is it possible to move the content of my local server to this one?
The server is working and states are saved, but my docker volume folder is still empty?!
So i dont know where i should move the data

**Comment by Blueforcer** [comment]
ok, i had an error in the volumes.. sorry

**Comment by Okynawah** [comment]
For any one wondering: `</path/to/your/palworld/folder>/Pal/Saved/SaveGames`
Just copy the folder to your local game and it should work (need confirmation here)

**Comment by Blueforcer** [comment]
moving the world works.
But I'm unable to copy the player data. 
I need to create a new character, but the I have no access my base because I'm not in that guild.

Renaming the original player save data doesn't help :(

Any idea?

**Comment by malobaidan** [comment]
> moving the world works.
> But I'm unable to copy the player data. 
> I need to create a new character, but the I have no access my base because I'm not in that guild.
> 
> Renaming the original player save data doesn't help :(
> 
> Any idea?

Im not at my pc now but you need to go to the server save/savegames/0/

You will find file with numbers name copy it and REPLACE the old folder in the new server after that find GameServerSetting.ini and edit the DetecatedServerName to the old name

I hope its clear

**Comment by thijsvanloef** [comment]
Since I believe this is not an issue with my code, I'm converting this issue to a discussion
